src/Components/Pages/StoppingPower.js

```diff
@@ -55,21 +66,35 @@ class StoppingPowerComponent extends React.PureComponent {
         return { xAxis, yAxis, method }
     }
 
+    async onXAxisChange(xAxis){
+        const dataSeries = await Promise.all(this.state.dataSeries.map(async ({data,metadata})=>{
+            const {color, isShown, index, name} = data
+            const newData = Object.assign(
+                {color, isShown, index, name}
+                , await this.wrapper.getDataSeries(metadata, xAxis===1)
```

Comment:
It is hard to guess what `xAxis===1` means. Is it log scale or linear scale. `1` is a bit meaningless.

Reply:
Done